projects/extension/package.json

```
@@ -24,14 +24,17 @@
   },
   "keywords": [],
   "devDependencies": {
+    "@jest/globals": "^27.5.1",
     "jest": "^27.5.1",
+    "jest-chrome": "^0.7.2",
     "web-ext": "^6.7.0"
   },
   "dependencies": {
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/styles": "^4.11.4",
     "@polkadot/rpc-provider": "^7.10.1",
+    "@substrate/connect": "0.6.0",
```

Not that it really matters but:
```suggestion
    "@substrate/connect": "^0.6.0",
```